### DISCUSSION TITLE
User identification and cross-device gaming

### DISCUSSION THREAD

**Discussion by CrsiX** [hopfenspace/runciv#1] [Ideas]
Current idea: a user is identified by username and password, so that playing on multiple devices works seamlessly (not at the same time for obvious reasons, but handling one round on a desktop and handling the other on a mobile device should work).

**Comment by touhidurrr** [comment]
Already implemented, see: https://github.com/yairm210/Unciv/pull/8716 https://github.com/yairm210/Unciv/pull/8770 https://github.com/yairm210/Unciv/pull/8779 https://github.com/yairm210/Unciv/pull/8790
